core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/settings/SettingsViewModel.kt

```diff
@@ -117,7 +122,13 @@ class SettingsViewModel(
 
     private val settings: Settings by instance()
 
+    private val sendAppFileUseCase: SendAppFileUseCase by instance()
```

Comment:
This would **crash** all non-Android platforms. This must be changed to use instanceOrNull. If it is null, then the share option should not be visible.